src/store.cc

```diff
@@ -2071,12 +2076,18 @@ StoreEntry::describeTimestamps() const
 }
 
 bool
-StoreEntry::collapsingInitiator() const
+StoreEntry::hittingRequiresCollapsing() const
 {
-    if (!publicKey())
-        return false;
-    return EBIT_TEST(flags, ENTRY_FWD_HDR_WAIT) ||
-           (hasTransients() && !hasMemStore() && !hasDisk());
+    return EBIT_TEST(flags, ENTRY_REQUIRES_COLLAPSING);
```

Comment:
Please inline this method for performance sake. I believe Store.h already has access to `EBIT_TEST` and `enum.h`.

Reply:
Done.